bin/pull-image

```diff
@@ -1,13 +1,20 @@
 #! /usr/bin/env bash
 
-# DOC: Pull the latest development environment docker image from Docker Hub unless USE_LOCAL_CIVIFORM is set.
+# DOC: Pull the latest development environment docker image from Docker Hub
+# DOC: unless USE_LOCAL_CIVIFORM is set. May specify what to pull with flags
```

Comment:
You can only specify one right?  Include that here.

Reply:
Nope can specify multiple

Reply:
ohwait, crap yeah only one. thanks